apps/homeless-people-help-map/src/components/App.tsx

```diff
@@ -123,17 +125,34 @@ export const App = () => {
     return e;
   }, [districtFilter.expression, layerFilterFixedExpression]);
 
+  const handleSideBar = useCallback((value: boolean, changePrevious: boolean) => {
+    if (changePrevious) frameStateSideBar.current = value;
+    setSidebarVisible(value);
+  }, []);
+
   // close sidebar on mobile and open on desktop
   useEffect(() => {
     // from mobile to desktop
-    if (previousMobile !== false && isMobile === false) {
-      setSidebarVisible(true);
+    if (previousMobile == true && !isMobile) {
```

Comment:
for consistency, either (with triple `=`)

```
    if (previousMobile === true && !isMobile) {
```

or perhaps just

```suggestion
    if (previousMobile && !isMobile) {
```